### ISSUE TITLE
Where can i find the published paper for this, please? 

### ISSUE THREAD

**Issue by Trevahok** [junhao12131/blaze#14]
I would like to work on this for another aspect, is this possible to find on a peer reviewed journal?

**Comment by jl2922** [comment]
No, I haven't published it and have left academia. You are welcome to work on it.

**Comment by Trevahok** [comment]
Thanks a lot. Would you be open to questions if something arises ? If so, where can i contact you at?
